persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/TestEntity.scala

```diff
@@ -43,9 +43,9 @@ object TestEntity {
     override def aggregateTag: AggregateEventShards[Evt] = Evt.aggregateEventShards
   }
 
-  final case class Appended(element: String) extends Evt
+  final case class Appended(entityId: String, element: String) extends Evt
```

Comment:
and let's do this for scaladsl first, and then we can see if we want to add it in javadsl also (in compatible way)